features/admin.core.v1/constants/app-constants.ts

```diff
@@ -507,25 +507,38 @@ export class AppConstants {
     /**
      * Route ids that are enabled in an organization.
      */
-    public static readonly ORGANIZATION_ENABLED_ROUTES: string[] = [
-        "gettingStarted",
-        "identityProviders",
-        "users",
-        "organizations",
-        "groups",
-        "roles",
-        "userRoles",
-        "applications",
-        "emailTemplates",
-        "smsTemplates",
-        "governanceConnectors",
-        "branding",
-        "consoleSettings",
-        "apiResources",
-        "approvals",
-        "workflows",
-        "flows"
-    ];
+    public static readonly ORGANIZATION_ENABLED_ROUTES: Record<string, string> = {
```

Comment:
Do we need this variable if we can take the same value from deployment config?

Reply:
This was added as a fallback in case the deployment config was failed to resolve due to some reason